testsuite/integration/elytron-oidc-client/src/test/java/org/wildfly/test/integration/elytron/oidc/client/deployment/OidcWithDeploymentConfigTest.java

```diff
@@ -350,6 +368,39 @@ public static WebArchive createOpenIDWithMissingSecretHmacSigningAlgorithm() {
                 .addAsWebInfResource(OidcWithDeploymentConfigTest.class.getPackage(), MISSING_SECRET_WITH_HMAC_ALGORITHM_FILE, "oidc.json");
     }
 
+    @Deployment(name = FORM_WITH_OIDC_EAR_APP, managed = false, testable = false)
+    public static Archive<?> createFormWithOidcDeployment() {
```

Comment:
Do you think it would also be possible to add a test case to `OidcWithSubsystemConfigTest`? The main difference with the subsystem config tests is that instead of specifying the oidc.json file in the deployment and specifying the `OIDC` auth-method in the deployment, a `secure-deployment` is configured in the `elytron-oidc-client` subsystem. For most cases, we try to add tests to both `OidcWithDeploymentConfigTest` and `OidcWithSubsystemConfigTest` to make sure both methods of configuring OIDC work for the specific use case.

Reply:
done